platform/windows/detect.py

```diff
@@ -180,37 +180,37 @@ def configure_msvc(env, manual_msvc_config):
 
     if env["target"] == "release":
         if env["optimize"] == "speed":  # optimize for speed (default)
-            env.Append(CCFLAGS=["/O2"])
+            env.Append(CCFLAGS=["/O2", "/MP"])
         else:  # optimize for size
-            env.Append(CCFLAGS=["/O1"])
+            env.Append(CCFLAGS=["/O1", "/MP"])
         env.Append(LINKFLAGS=["/ENTRY:mainCRTStartup"])
         env.Append(LINKFLAGS=["/OPT:REF"])
 
     elif env["target"] == "release_debug":
         if env["optimize"] == "speed":  # optimize for speed (default)
-            env.Append(CCFLAGS=["/O2"])
+            env.Append(CCFLAGS=["/O2", "/MP"])
         else:  # optimize for size
-            env.Append(CCFLAGS=["/O1"])
+            env.Append(CCFLAGS=["/O1", "/MP"])
         env.AppendUnique(CPPDEFINES=["DEBUG_ENABLED"])
         env.Append(LINKFLAGS=["/OPT:REF"])
 
     elif env["target"] == "debug":
-        env.AppendUnique(CCFLAGS=["/Z7", "/Od", "/EHsc"])
+        env.AppendUnique(CCFLAGS=["/Z7", "/Od", "/EHsc", "/MP"])
         env.AppendUnique(CPPDEFINES=["DEBUG_ENABLED"])
-        env.Append(LINKFLAGS=["/DEBUG"])
+        env.Append(LINKFLAGS=["/DEBUG:fastlink"])
 
     env.Append(LINKFLAGS=["/SUBSYSTEM:WINDOWS"])
 
     if env["debug_symbols"] == "full" or env["debug_symbols"] == "yes":
         env.AppendUnique(CCFLAGS=["/Z7"])
-        env.AppendUnique(LINKFLAGS=["/DEBUG"])
+        env.AppendUnique(LINKFLAGS=["/DEBUG:fastlink"])
 
     ## Compile/link flags
 
     env.AppendUnique(CCFLAGS=["/MT", "/Gd", "/GR", "/nologo"])
     if int(env["MSVC_VERSION"].split(".")[0]) >= 14:  # vs2015 and later
         env.AppendUnique(CCFLAGS=["/utf-8"])
-    env.AppendUnique(CXXFLAGS=["/TP"])  # assume all sources are C++
+    env.AppendUnique(CXXFLAGS=["/TP", "/MP"])  # assume all sources are C++
```

Comment:
Isn't it redundant with applying it to `CCFLAGS` (which apply to both C and C++ sources)?